.gitignore

```diff
@@ -31,6 +31,7 @@ bower_components
 
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
+dist
```

Comment:
As the build is placed in the `build` directory, I believe we don't need this. Or am I missing something?